packages/react/src/components/Modal/ModalTrigger.tsx

```diff
@@ -3,26 +3,22 @@ import { forwardRef, useContext } from 'react';
 import type { ComponentPropsWithRef } from 'react';
 
 import { Button } from '../Button/Button';
-
-import { ModalContext } from './ModalRoot';
+import { Context } from './ModalContext';
 
 export type ModalTriggerProps = ComponentPropsWithRef<typeof Button>;
 
 export const ModalTrigger = forwardRef<HTMLButtonElement, ModalTriggerProps>(
-  ({ asChild, ...rest }, ref) => {
-    const { modalRef, open } = useContext(ModalContext);
+  function ModalTrigger({ asChild, ...rest }, ref) {
+    const contextRef = useContext(Context);
     const Component = asChild ? Slot : Button;
 
     return (
       <Component
-        ref={ref}
-        onClick={() => modalRef?.current?.showModal()}
-        aria-expanded={open}
```

Comment:
Do we remove this since focus in moved from the button to the dialog, meaning this is not relevant for the user?

Reply:
Jupp, the `aria-expanded` will never be read by any screen reader ☺️